### DISCUSSION TITLE
How to read the Distance to Arrival field

### DISCUSSION THREAD

**Discussion by lawrencemaroun** [alandtse/tesla#802] [Q&A] [answered]
![image](https://github.com/alandtse/tesla/assets/31968943/0101db60-4caa-47df-860f-65fe014f3b23)
When I look at the history of the Distance to Arrival field, it looks like there are Min, Mean, and Max attributes within.

![image](https://github.com/alandtse/tesla/assets/31968943/208b9c23-e968-4ae0-a494-1748433e687c)

However, when I try to write a if-then statement based on the numeric state of the DTA field, the only attribute option is called "Attribution". How would I write an if-then statement if I want to take an action if the DTA is below a certain value? Also, what do the Min, Mean, and Max mean exactly? Why not just show the exact distance at the time of polling?

**Comment by skipishere** [comment, marked as answer]
The min and max on that graph are part of home assistant behaviour. You will find this with other sensors that change often as well, such as temperatures ![Screenshot_20231221-200919.png](https://github.com/alandtse/tesla/assets/26724777/dbe98ff1-6430-421d-96fe-ba9680234bd0)

**Reply by lawrencemaroun** [reply]
Thanks! I haven't seen that before, none of my other fields do this. Including my temperature field for the thermostat.

**Reply by skipishere** [reply]
Trends to appear on ones that update more frequently, my thermostats for example don't but my pi's temperature does. (Caught me by surprise when they first introduced it as well)
